### PR TITLE
pep8 fixes

### PR DESCRIPTION
**Pull request by galstrom21**

### CODE REVIEW

File: webapp/filters.py [hunk 1]
```diff
@@ -209,7 +230,7 @@ def dotty(self, fd):
         lhs_id = 'x'
         rhs_id = 'x'
 
-        if type(self.lhs) == type(3) or type(self.lhs) == type('x'):
+        if isinstance(self.lhs, 3) or isinstance(self.lhs, 'x'):
```
Comment (rpedde) [inline webapp/filters.py]: isinstance(self.lhs, int) or isinstance(self.rhs, str)

File: webapp/filters.py [hunk 2]
```diff
@@ -217,7 +238,7 @@ def dotty(self, fd):
             lhs_id = id(self.lhs)
             self.lhs.dotty(fd)
 
-        if type(self.rhs) == type(3) or type(self.rhs) == type('x'):
+        if isinstance(self.rhs, 3) or isinstance(self.rhs, 'x'):
```
Comment (rpedde) [inline webapp/filters.py]: isinstance(self.lhs, int) or isinstance(self.rhs, str)

**Comment by rpedde** [comment]
pleasefix.

**Comment by galstrom21** [comment]
self merging cause ron hates puppies

**Pull request merged**